CMakeLists.txt

```diff
@@ -103,20 +104,26 @@ if(NOT MSVC AND NOT XCODE_VERSION)
   endif()
 endif()
 
-# Options relating to MATLAB wrapper
-# TODO: Check for matlab mex binary before handling building of binaries
-option(GTSAM_INSTALL_MATLAB_TOOLBOX      "Enable/Disable installation of matlab toolbox"  OFF)
-set(GTSAM_PYTHON_VERSION "Default" CACHE STRING "The version of Python to build the wrappers against.")
 
 # Check / set dependent variables for MATLAB wrapper
-if(GTSAM_INSTALL_MATLAB_TOOLBOX AND GTSAM_BUILD_TYPE_POSTFIXES)
-    set(CURRENT_POSTFIX ${CMAKE_${CMAKE_BUILD_TYPE_UPPER}_POSTFIX})
-endif()
+if(GTSAM_INSTALL_MATLAB_TOOLBOX)
+    find_package(Matlab COMPONENTS MEX_COMPILER REQUIRED)
```

Comment:
I think you should probably remove the relevant code in `cmake/GtsamMatlabWrap`?

Reply:
I don't think there is any related code in `GtsamMatlabWrap`? I may be wrong though. My understanding is that the code in `GtsamMatlabWrap` is primarily for wrapping and I didn't want to mess with it for minor things.